components/TextFields/src/ContainedInputView/MDCFilledTextField.m

```diff
@@ -48,6 +48,17 @@ - (void)commonMDCFilledTextFieldInit {
   self.borderStyle = UITextBorderStyleNone;
 }
 
+#pragma mark UITextField Overrides
+
+- (void)setBorderStyle:(UITextBorderStyle)borderStyle {
```

Comment:
We need to mention this change to the filled text field in the PR description.

Ideally we would have done this Filled text field change in a separate PR

Reply:
I didn't really see the benefit of opening a new PR for this change. I only saw it as another thing to get approved haha. In any case, I'm going to remove this change, since we're agreeing on marking it as unavailable instead of this.

Reply:
Of course now there's just a different change to the filled textfield--marking it as unavailable. I can open a different PR with that change if you'd like, or keep it in this one.